geoportal/c2cgeoportal_geoportal/views/theme.py

```diff
@@ -189,9 +189,10 @@ async def _wms_getcap(self, ogc_server, preload=False):
         def build_web_map_service(ogc_server_id):
             del ogc_server_id  # Just for cache
 
+            version = urllib.parse.parse_qs(urllib.parse.urlsplit(url).query)['VERSION'][0]
```

Comment:
We call `urlsplit` and `parse_qs` 3 times on the same URL ...
Looks that we should refactor it a little on master...